src/pci_lmt/results.py

```diff
@@ -64,6 +67,21 @@ def to_csv(self) -> ty.Tuple[str, str]:
 
 
 class Reporter:
+    def __init__(self):
+        pass
+
+    def start_run(self, host: HostInfo, version: str) -> None:
```

Comment:
i strongly suggest that you use a contextmanager here; see `run.scope() / test.scope()` in ocptv; this way, other code in pci_lmt wont forget to call the end methods, for either runs or steps

Reply:
Thanks for the suggestion and pointers. I've incorporated your comments.